spec/models/info_request_spec.rb

```diff
@@ -307,32 +307,30 @@
         to eq('rejected for testing')
     end
 
-    context 'emailing the request owner' do
+    describe 'notifying the request owner' do
 
-      it 'emails the user that a response has been received' do
+      it 'notifies the user that a response has been received' do
         info_request = FactoryGirl.create(:info_request)
         email, raw_email = email_and_raw_email
+        user = info_request.user
+
+        # Without this, the user retrieved in the model is not the same one
+        # in memory as this one, so we can't put expectations on it
+        allow(info_request).to receive(:user).and_return(user)
+        expect(user).
```

Comment:
With the mocking still in place this can work, i.e.:

```diff
-        user = info_request.user

         # Without this, the user retrieved in the model is not the same one
         # in memory as this one, so we can't put expectations on it
+        allow(info_request).to receive(:user).and_return(info_request.user)
+        expect(info_request.user).
```

Is that what you meant? I'm happy to make that change if it's clearer, I've pushed it as 7cd9b3995f1b1cb1072752a10c9150b414eb9187